src/frontend/context/AppContext.tsx

```diff
@@ -29,13 +34,69 @@ interface IProps {
 
 const [AppProvider, useApp] = createUseContext(({ autentisertSaksbehandler }: IProps) => {
     const [autentisert, settAutentisert] = React.useState(true);
+    const [appVersjon, settAppVersjon] = useState('');
     const [ressurserSomLaster, settRessurserSomLaster] = React.useState<string[]>([]);
 
     const [innloggetSaksbehandler, settInnloggetSaksbehandler] = React.useState(
         autentisertSaksbehandler
     );
     const [modal, settModal] = React.useState<IModal>(initalState);
 
+    const verifiserVersjon = () => {
+        axiosRequest<string, void>({
+            url: '/version',
+            method: 'GET',
+        }).then((versjon: Ressurs<string>) => {
+            if (versjon.status === RessursStatus.SUKSESS) {
+                if (appVersjon !== '' && appVersjon !== versjon.data) {
+                    settModal({
+                        tittel: 'Løsningen er utdatert',
+                        innhold: () => {
+                            return (
+                                <>
+                                    <Normaltekst>
```

Comment:
Her skal det inn et ikon, men det var ikke eksportert så jeg fant ikke svg'en for det.